docs/src/components/landing/ToolpadFeaturesSwitcher.tsx

```diff
@@ -3,7 +3,9 @@ import dynamic from 'next/dynamic';
 import Box from '@mui/material/Box';
 import Stack from '@mui/material/Stack';
 import Typography from '@mui/material/Typography';
+import Link from '@mui/material/Link';
```

Comment:
This page lives in the same Next.js app as the docs, so it's an internal link. We could avoid a full navigation if we use the Next.js router. Is it possible to instead use
```suggestion
import { Link } from '@mui/docs/Link';
```